django/db/backends/oracle/schema.py

```diff
@@ -42,6 +44,41 @@ def quote_value(self, value):
         else:
             return str(value)
 
+    def create_model(self, model):
+        super().create_model(model)
+        if model._meta.db_table_comment:
+            self.execute(
+                self.sql_alter_table_comment
+                % {
+                    "table": model._meta.db_table,
+                    "comment": self.quote_comment_value(model._meta.db_table_comment),
+                },
+                [],
+            )
+        for field in model._meta.local_fields:
+            if field.db_comment:
+                self.execute(
+                    self.sql_alter_column_comment
+                    % {
+                        "table": model._meta.db_table,
+                        "column": field.column,
+                        "comment": self.quote_comment_value(field.db_comment),
+                    },
+                    [],
+                )
+
+    def add_field(self, model, field):
+        super().add_field(model, field)
+        self.execute(
+            self.sql_alter_column_comment
+            % {
+                "table": model._meta.db_table,
+                "column": field.column,
+                "comment": self.quote_comment_value(field.db_comment),
+            },
+            [],
+        )
```

Comment:
This looks the same for all backends that support this? Can we not just check the `supports_comments` flag in base class' `add_field()` method?